ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -571,4 +573,11 @@ QtObject {
     function hex2Eth(value) {
         return globalUtils.hex2Eth(value)
     }
+
+    readonly property Connections connections: Connections {
```

Comment:
Since we may want to connect to more than one module in a store:
```suggestion
    readonly property Connections communitiesModuleConnections: Connections {
```